.gitignore

```diff
@@ -26,3 +26,7 @@ a.out
 /outputGotten.txt
 /testresults
 /tests/emscripten/*.js
+
+.gitignore
+*.elf
+*jseb*
```

Comment:
.gitignore is a file used by git to tell which files shouldn't be committed to the repository. Usually it is build artifacts or test artifacts. It should be for general use by everybody. So
- `.gitignore` file shouldn't be ignored
- `*jseb*` is strictly personal use, you should manage any personal files/folders by yourself

Reply:
i will revert and put my personal configuration in another file (i see i could use ~/.gitignore for example)